packages/scraper-tools/src/browser_tools.ts

```diff
@@ -1,10 +1,18 @@
 import { Actor } from 'apify';
 import log from '@apify/log';
-import type { Page } from 'puppeteer';
+import type { CommonPage } from '@crawlee/browser-pool';
 import { inspect } from 'util';
 import { RESOURCE_LOAD_ERROR_MESSAGE, SNAPSHOT } from './consts';
 import { createRandomHash } from './tools';
 
+// TODO
```

Comment:
TODO what? lets comment whats missing

Reply:
Pretty much. It's here because it was puppeteer only, now it has to support both. I just wasn't sure if that was exactly the right thing to do. Will remove todo